### ISSUE TITLE
[Bug]:Data duplication is displayed in the show statement for common tenants.

### ISSUE THREAD

**Issue by tianyahui-python** [matrixorigin/matrixone#7913]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):db3dec1d612b78d6989eacfbe81a4f3a3d3dd3df
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Under common tenant：
execute sql:

create database test_db;
use test_db;
create table test_table(
col1 int,
col2 varchar
);
create view test_view as select * from test_table;
show create database test_db;
show create table test_table;
show triggers;
show procedure status;
show config;
show events;
show plugins;
show profiles;
show privileges;
show processlist;
show tables;

<img width="618" alt="截屏2023-02-08 17 35 49" src="https://user-images.githubusercontent.com/39303074/217492440-4ac50c0d-bc30-4b7e-8749-b7298f5dd6d8.png">

**Comment by aressu1985** [comment]
it should be the same as issue #7897

**Comment by daviszhen** [comment]
It was fixed at pr #7914

**Comment by aressu1985** [comment]
fxied: 4a2132a54f9e643346878e0be991c47dce965d9b
